README.md

```diff
@@ -139,6 +139,16 @@ and run `vagrant up`.
 [go]: https://golang.org/dl/
 [kubeflow]: https://github.com/kubeflow/kubeflow
 
+
+## On platform testing
+
+Any push to an open PR that also has the auto-deploy label on the PR This allows developers to opt-in to on-platform testing. For example, when you need to build in github and test on platform (or want someone else to be able to pull your image):
```

Comment:
I feel like there is something missing in this text. "... auto-deploy label on the PR This allows ..." feels off

Reply:
Would you have a better suggestion since it's here 
![image](https://user-images.githubusercontent.com/14837042/207602056-429a3e62-07df-43af-9053-d3c300201023.png)